File: trunk/quests/burningwood/Azdalin.pl
#Zone: Burningwood  NPC: Azdalin (87152)
#by: Qadar

sub EVENT_COMBAT {
  my $random_result = int(rand(60));
  if($combat_state == 1) {
    quest::say("Foolish adventurer!!  You shall die.  You are nothing to Azdalin. ");
    if($random_result <=20) {
      quest::say("${class}s like you always bring out the worst in me!");
    }
    elsif($random_result <=40){
      quest::say("I really hate ${class}s like you!");
    }
    else{
      quest::say("${class}s like you are an affront to my senses!");
    }
  }
}

File: burningwood/Azdalin.lua
function event_combat(e)
	local random_result = math.random(60);
	
	if(e.joined) then
		e.self:Say("Foolish adventurer!!  You shall die.  You are nothing to Azdalin. ");
		if(random_result <=20) then
			e.self:Say("" .. e.other:GetClassName() .. " like you always bring out the worst in me!");
		elseif(random_result <=40) then
			e.self:Say("I really hate " .. e.other:GetClassName() .. " like you!");
		else
			e.self:Say("" .. e.other:GetClassName() .. " like you are an affront to my senses!");
		end
	end
end
